doc/source/templates/testing/docker/04_finetuning_llms_with_deepspeed/Dockerfile

```diff
@@ -1,12 +1,15 @@
 # Dockerfile used to create the docker image for `04_finetuning_llms_with_deepspeed`.
-FROM anyscale/ray:2.7.1-py310-cu121
+FROM anyscale/ray:2.8.0-py310-cu121
```

Comment:
we should probably wait for the ray 2.8.1 release

Reply:
Sure. We can hand the docker image to folks who really need it until then.

Reply:
@justinvyu Is there a particular reason for that?
Because I've verified that this thing works already and don't want to go through that process again if 2.8.1 does not bring anything relevant to the template, right?